Placing a custom block causes an ArrayIndexOutOfBoundsException in MixinBlock_BlockCapturing on initializeBlockCapturingState

**I am currently running**
<!-- If you don't use the latest version, please tell us why. -->
- SpongeForge version: spongeforge-1.12.2-2611-7.1.0-BETA-2970.jar
- Forge version: Forge Mod Loader version 14.23.2.2635
- SpongeVanilla version: NO
- Java version: 8_186
- Operating System: DEBIAN

<!-- Please include ALL mods/plugins you had installed when your issue happened, you can get a list of 
        your mods and plugins by running "/sponge plugins" and/or "/sponge mods" -->
- Plugins/Mods: SpongeForge and DynmapForge, rest are private

<!-- Please include as much information as possible. For the description, assume we have no idea how 
        mods work, be as detailed as possible and include a step by step reproduction. It is recommended 
        you try to reproduce the issue you are having yourself with as few mods as possible. -->
**Issue Description**
I add custom blocks with Forge. Then I go ingame on forge server and place custom block. It works. Then I add SpongeForge and place custom block I get evil exception (see below). Doesnt make sense tbh because the same method that is used to get the id I use from forge and it is "gss-decor:statue_jupiter" which when split with `:` should have length 2 which it apparently doesnt have when sponge calls it. Im confused sorry.

Relevant Mixins
https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/mixin/blockcapturing/MixinBlock_BlockCapturing.java#L55
https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/mixin/core/block/MixinBlock.java#L169

Calling this from Forge gets "gss-decor:statue_jupiter" (its used from the Mixin):
```java
Block.REGISTRY.getNameForObject((Block) (Object) this).toString();
```

But somehow Sponge fails at `ids[1]`:

```java
String[] ids = this.getId().split(":");
String modId = ids[0];
String name = ids[1];
```

Exception
```
[22:31:23] [Server thread/ERROR] [net.minecraft.server.MinecraftServer]: Encountered an unexpected exception
net.minecraft.util.ReportedException: Exception while ticking a block
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:760) ~[MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396) ~[nz.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_162]
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at net.minecraft.block.Block.initializeBlockCapturingState(Block.java:3705) ~[aow.class:?]
	at org.spongepowered.common.event.tracking.TrackingUtil.checkAndAssignBlockTickConfig(TrackingUtil.java:335) ~[TrackingUtil.class:1.12.2-2611-7.1.0-BETA-2970]
	at org.spongepowered.common.event.tracking.TrackingUtil.updateTickBlock(TrackingUtil.java:276) ~[TrackingUtil.class:1.12.2-2611-7.1.0-BETA-2970]
	at net.minecraft.world.WorldServer.redirect$onUpdateTick$zll000(WorldServer.java:2248) ~[oo.class:?]
	at net.minecraft.world.WorldServer.func_72955_a(WorldServer.java:737) ~[oo.class:?]
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:223) ~[oo.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:754) ~[MinecraftServer.class:?]
	... 4 more
```

<!-- If you are a developer, please attach a short and self contained code snippet (http://sscce.org) to 
        replicate your issue. Any relevant log files and snippets should be uploaded to gist.github.com!
        Don't use hastebin/pastebin or other similar sites, as they have a history of quickly
        deleting files before we can look at them. -->

--- a/src/main/java/org/spongepowered/common/mixin/api/mcp/advancements/AdvancementMixin_API.java
+++ b/src/main/java/org/spongepowered/common/mixin/api/mcp/advancements/AdvancementMixin_API.java
@@ -98,7 +98,7 @@ public class AdvancementMixin_API implements org.spongepowered.api.advancement.A
     }
 
     @Override
-    public String getId() {
+    public final String getId() {
         checkState(SpongeImplHooks.isMainThread());
         return ((AdvancementBridge) this).bridge$getId();
     }

--- a/src/main/java/org/spongepowered/common/mixin/api/mcp/block/BlockMixin_API.java
+++ b/src/main/java/org/spongepowered/common/mixin/api/mcp/block/BlockMixin_API.java
@@ -67,7 +67,7 @@ public abstract class BlockMixin_API implements BlockType {
     @Shadow public abstract BlockStateContainer getBlockState();
 
     @Override
-    public String getId() {
+    public final String getId() {
         return this.getNameFromRegistry();
     }
 

--- a/src/main/java/org/spongepowered/common/mixin/api/mcp/enchantment/EnchantmentMixin_API.java
+++ b/src/main/java/org/spongepowered/common/mixin/api/mcp/enchantment/EnchantmentMixin_API.java
@@ -63,7 +63,7 @@ public abstract class EnchantmentMixin_API implements EnchantmentType {
     @Nullable private String api$id;
 
     @Override
-    public String getId() {
+    public final String getId() {
         if (this.api$id == null || this.api$id.isEmpty()) {
             final ResourceLocation id = REGISTRY.getNameForObject((Enchantment) (Object) this);
             if (id != null) {

--- a/src/main/java/org/spongepowered/common/mixin/api/mcp/item/ItemMixin_API.java
+++ b/src/main/java/org/spongepowered/common/mixin/api/mcp/item/ItemMixin_API.java
@@ -54,7 +54,7 @@ public abstract class ItemMixin_API implements ItemType {
     @Nullable private org.spongepowered.api.item.inventory.ItemStack propertyItemStack;
 
     @Override
-    public String getId() {
+    public final String getId() {
         final ResourceLocation resourceLocation = SpongeImplHooks.getItemResourceLocation((Item) (Object) this);
         checkState(resourceLocation != null, "Attempted to access the id before the Item is registered.");
         return resourceLocation.toString();

--- a/src/main/java/org/spongepowered/common/mixin/api/mcp/world/biome/BiomeMixin_API.java
+++ b/src/main/java/org/spongepowered/common/mixin/api/mcp/world/biome/BiomeMixin_API.java
@@ -66,7 +66,7 @@ public abstract class BiomeMixin_API implements BiomeType {
     }
 
     @Override
-    public String getId() {
+    public final String getId() {
         return ((BiomeBridge) this).bridge$getId();
     }
 

